api/policies/abUtils.js

```diff
@@ -65,10 +65,17 @@ module.exports = (req, res, next) => {
                errors.push({ message: errMsg, packet: data });
                return;
             }
-            // Log to Prometheus server
-            MetricManager.logSocketPayload({
-               event: d.event,
-               data: d.data,
+
+            let userSockets = sails.io.sockets.in(d.room);
+            // {array} of all the individual sockets in the room we are 
+            // sending a message to.
+
+            (userSockets || []).forEach((s) => {
```

Comment:
There is a error `userSockets.forEach is not a function`.

Should use this instead `Object.keys(userSockets.sockets).forEach((s) => {` ?

Reply:
you're right!